src/examples/service-unavailable/service-temporarily-unavailable-welsh/index.njk

```diff
@@ -7,11 +7,8 @@ layout: layout-example.njk
   text: 'Mae’n ddrwg gennym, ond nid yw’r gwasanaeth hwn ar gael ar hyn o bryd'
 }) }}
 
-<p class="govuk-body">Byddwch yn gallu defnyddio’r gwasanaeth o 09:00 ar Ddydd Llun 19 Tachwedd 2018 ymlaen.</p>
+<p class="govuk-body">Byddwch yn gallu defnyddio’r gwasanaeth o 09:00 ar Ddydd Llun 19 Tachwedd 2018.</p>
 
-<p class="govuk-body">Gallwch:</p>
+<p class="govuk-body">Nid ydym wedi cadw’ch atebion. Pan fydd y gwasanaeth ar gael, bydd yn rhaid i chi ddechrau eto.</p>
 
-<ul class="govuk-list govuk-list--bullet">
-  <li><a class="govuk-link" href="#">newid manylion TAW eraill</a></li>
-  <li><a class="govuk-link" href="#">cysylltu â Gwasanaeth Cwsmeriaid Cymraeg CThEM</a> os oes angen i chi wneud newidiadau i’ch cais neu siarad â rhywun am eich credydau treth</li>
-</ul>
+<p class="govuk-body"><a href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/welsh-language-helplines">Cysylltwch â Gwasanaeth Cwsmeriaid Cymraeg CThEM</a> os oes angen i chi wneud newidiadau i’ch hawliad neu siarad â rhywun am eich credydau treth.</p>
```

Comment:
I'm guessing this updates the formatting to be the same as the English version?

Reply:
Yes, the previous welsh translation had actually been for a different example